template/build.gradle.kts

```
@@ -54,15 +49,16 @@ detekt {
 
     ignoredBuildTypes = listOf("release")
     ignoredFlavors = listOf("production")
+}
 
+tasks.withType<io.gitlab.arturbosch.detekt.Detekt>().configureEach {
+    jvmTarget = JavaVersion.VERSION_11.toString()
     reports {
         xml {
-            enabled = true
-            destination = file("build/reports/detekt.xml")
+            outputLocation.set(file("build/reports/detekt.xml"))
```

`Archive Detekt reports on template` in `run_detekt_and_unit_tests.yml` expects path `template/build/reports/detekt/`

```suggestion
            outputLocation.set(file("build/reports/detekt/detekt.xml"))
```

Shall we re-factor it like below? Because the default is `template/build/reports/detekt/` anyways
 
```
tasks.withType<io.gitlab.arturbosch.detekt.Detekt>().configureEach {
    jvmTarget = JavaVersion.VERSION_11.toString()
}
```